### ISSUE TITLE
Fishing Rod and Slime block

### ISSUE THREAD

**Issue by Kixot14** [NoCheatPlus/Issues#192]
Can you please fixing 2 thing ??:

## 2. If you stay on the piston with slime block and when you activate it, slime must push up you at 10-15 blocks. Very cool gameplay moment. But NCP block this action :[
- failed SurvivalFly

At the moment of test i use 
  # NCP build #985
  # Spigot build #847

**Comment by asofold** [comment]
Slime blocks: https://github.com/NoCheatPlus/Issues/issues/5

For fishing rod i'd prefer to have a debug log (https://github.com/NoCheatPlus/Docs/wiki/Debugging#on-the-fly-debug-output-for-individual-players).

**Comment by Kixot14** [comment]
Here is the output

git-Spigot-8a048fe-5ff377a ~MC: 1.9.4~
  detected: 1.9.4
#### NoCheatPlus ####
Plugin: 3.14.0-SNAPSHOT-sMD5NET-b985
MCAccess: 1.9.4 / Spigot-CB-1.9_R2
Features:
  blocks: BlocksMC1_4 | BlocksMC1_5 | BlocksMC1_6_1 | BlocksMC1_7_2 | BlocksMC1_8 | BlocksMC1_9
  checks: FastConsume | Gutenberg | AttackFrequency | FlyingFrequency | KeepAliveFrequency
  defaults: pvpKnockBackVelocity
  packet-listeners: UseEntityAdapter | MovingFlying | OutgoingPosition | KeepAliveAdapter
Hooks: AllViolations~NCP~ 1.0
#### Related Plugins ####
ProtocolLib v4.0.1

**Comment by asofold** [comment]
Thanks a lot. So we'll have to see what changed velocity-wise, or if we need a debug log. I'll assume there is no plugin involved for fishing rods with pvp.

**Comment by asofold** [comment]
Has this changed with the latest development builds of NCP, is anyone able to reproduce these kind of issues with fishing rods and pvp?

**Comment by RoboMWM** [comment]
> I'll assume there is no plugin involved for fishing rods with pvp.

Correct, but there is a PlayerFishEvent

**Comment by asofold** [comment]
@RoboMWM Yes, with that comment i was having in mind the possibility of plugins adding extra velocity or altering velocity. 

In fact we should know / find out if there is a velcocity event, if there is an outgoing velocity packet or if the dragging is client side for player vs. player (and for which Minecraft version applies which...).
